include/benchmark/benchmark.h

```diff
@@ -950,6 +950,9 @@ class Benchmark {
   // Equivalent to ThreadRange(NumCPUs(), NumCPUs())
   Benchmark* ThreadPerCpu();
 
+  // Specify the name of the benchmark
+  Benchmark* Name(const std::string& name);
```

Comment:
I think this should go all the way up, to just before `Benchmark* Arg(int64_t x);`